bin/MadGraph5_aMCatNLO/cards/production/13TeV/tWZ/template_run_card.dat

```diff
@@ -65,8 +65,7 @@
 # PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
 #***********************************************************************
   lhapdf	= pdlabel ! PDF set
-  $DEFAULT_PDF_SETS = lhaid
-
+  $$DEFAULT_PDF_SETS = lhaid
```

Comment:
Is there a reason to change this with two $s? It might not work. (same for $$DEFAULT_PDF_MEMBERS)

https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/Utilities/gridpack_helpers.sh

Reply:
The python template class, which I use in the script, uses $ as placeholder. Two $s escaped and are replaced with just one $, so it should be fine after running the script

Reply:
oh ok i see